.prettierignore

```diff
@@ -25,7 +25,7 @@ lib
 /contracts/dev-utils/generated-wrappers
 /contracts/dev-utils/generated-artifacts
 /packages/abi-gen/test/generated-test/generated-artifacts
-/packages/abi-gen/test/generated-test/generated-wrappers
```

Comment:
what do you think about calling it `generated-output`?

further, though, i think i'd like to see the whole path renamed! :smile:

how about `abi-gen/test-cli/generated-output`, alongside `test-cli/known-good-output`.

we have `abi-gen/test`, that's typically for unit tests. i suggest a separate folder `abi-gen/test-cli`, that's for testing the command-line interface. that folder can have its own `tsconfig`, and `abi-gen/test` can keep its own typical `tsconfig`.  apparently, testing the cli (verifying it's output against a known-good), gets pretty noisy in the filesystem and configuration, so i think it would be nice to wall it off in its own noisy folder and let the regular unit tests stand like they would for any other package.

seems good to me right now, but just brainstorming... what do you think? maybe we could let `output` slide for now and pick a better name later when we're doing a little more re-org of the tests anyways?